internal/resources/grafana/resource_alerting_rule_group.go

```diff
@@ -567,6 +575,14 @@ func unpackAlertRule(raw interface{}, groupName string, folderUID string, orgID
 		return nil, err
 	}
 
+	var missingSeriesEvalsToResolve int64
+	if val, ok := json["missing_series_evals_to_resolve"]; ok && val != nil {
+		intVal := val.(int)
+		if intVal >= 1 {
```

Comment:
Just a small nit to avoid panics in case of invalid values

```suggestion
		intVal, ok := val.(int)
		if ok && intVal >= 1 {
```